primitives/externalities/src/lib.rs

```diff
@@ -42,26 +40,40 @@ pub trait Externalities: ExtensionStore {
 	fn storage(&self, key: &[u8]) -> Option<Vec<u8>>;
 
 	/// Get storage value hash. This may be optimized for large values.
-	fn storage_hash(&self, key: &[u8]) -> Option<H256>;
+	fn storage_hash(&self, key: &[u8]) -> Option<Vec<u8>>;
 
 	/// Get child storage value hash. This may be optimized for large values.
-	fn child_storage_hash(&self, storage_key: ChildStorageKey, key: &[u8]) -> Option<H256>;
+	///
+	/// Returns an `Option` that holds the raw hash.
```

Comment:
what does raw hash means in this context ?

looking at the implementation:
```rust
	fn child_storage_hash(&self, storage_key: ChildStorageKey, key: &[u8]) -> Option<Vec<u8>> {
		self.child_storage(storage_key, key).map(|v| Blake2Hasher::hash(&v).encode())
```
it seems SCALE encoded hash as well no ?

```suggestion
	/// Returns an `Option` that holds the SCALE encoded hash.
```

Reply:
Yep.